index.html

```diff
@@ -203,6 +204,25 @@ <h1 class="largest-heading">Rescue<span>a pet</span></h1>
       </div>
      </div>
     </section>
+    <footer>
+     <div class="footer-heading">
+      <div class="main-footer-heading">
+      <p>
+         <span>Sample text. Click to select the text box. Click again or double</span>
+         <span>click to start editing the text.</span>
+       </p>
+     </div>
+     </div>
+     <div class="footer-item">
+      <div class="footer-description">
+      <p>
+        <span><a href="#">Template</a></span>
+        <span>created with</span>
+        <span><a href="a">Nicepage.</a></span>
```

Comment:
Nie jest to błąd, ale spana raczej daj do <a>

Reply:
Poprawione